rclcpp/src/rclcpp/time.cpp

```diff
@@ -251,5 +251,12 @@ operator+(const rclcpp::Duration & lhs, const rclcpp::Time & rhs)
   return Time(lhs.nanoseconds() + rhs.nanoseconds(), rhs.get_clock_type());
 }
 
+Time
+TIME_MAX()
+{
+  Time time_max(std::numeric_limits<int32_t>::max(), 999999999);
```

Comment:
This seems like it would be easier to read as a one line function. No need to name the temporary variable, it could just be returned.